opentreemap/treemap/js/src/mapPage/locationSearchUI.js

```diff
@@ -1,9 +1,15 @@
 "use strict";
 
 var $ = require('jquery'),
-    _ = require('lodash');
+    _ = require('lodash'),
+    Bacon = require('baconjs'),
+    reverse = require('reverse'),
+    config              = require('treemap/lib/config.js'),
+    BU = require('treemap/lib/baconUtils.js'),
+    boundarySelect = require('treemap/lib/boundarySelect');
 
 var dom = {
+    locationValue: '#boundary',
```

Comment:
This element is shared between custom area and search by location. It's consistent with the other `location*` dom property names.